### PR TITLE
MBS-6515: Option to show not yet reachable places

### PR DESCRIPTION
**Pull request by sh-csg**

### CODE REVIEW

File: templates/advancedsettings.mustache [hunk 1]
```diff
@@ -73,6 +73,14 @@
...
+                <div class="col">
+                    <label for="learningmap-showall">{{#str}} showall, learningmap {{/str}}</label>
+                </div>
+                <div class="col-sm">
+                    <input type="checkbox" id="learningmap-showall"/>
+                </div>
+            </div>
         </div>
     </div>
 </div>
```
Comment (PhMemmel) [inline templates/advancedsettings.mustache:86]: Newline fehlt

File: amd/src/placestore.js [hunk 2]
```diff
@@ -328,6 +330,20 @@ let placestore = {
...
     },
+    /**
+     * Sets showall attribute
+     * @param {boolean} value
+     */
+    setShowall: function(value) {
+        this.showall = value;
+    },
+    /**
+     * Returns the value of hover attribute
```
Comment (PhMemmel) [inline amd/src/placestore.js:341]: Copy and Paste mistake

File: templates/cssskeleton.mustache [hunk 3]
```diff
@@ -66,7 +72,12 @@
     {{/editmode}}
     {{/hidepaths}}
     {{^editmode}}
+    {{^showall}}
```
Comment (PhMemmel) [inline templates/cssskeleton.mustache:75]: Aren't lines 70-72 obsolete with this?

**Comment by PhMemmel** [comment]
Tested and reviewed, except from the two comments, fine from my point of view

File: templates/cssskeleton.mustache (hunk 3 above)
Comment (sh-csg) [reply templates/cssskeleton.mustache:75]: No, this is needed to hide the paths in "normal" mode.

Comment (PhMemmel) [reply templates/cssskeleton.mustache:73]: Latest commit improved readability, I overlooked `hidepaths` condition at first, sry

File: styles.css [hunk 4]
```diff
@@ -36,6 +36,7 @@
     position: relative;
     min-height: 100px;
     width: 100%;
+    max-width: 90vw;
```
[the remark this answers is not in the document]
Comment (PhMemmel) [reply styles.css:39]: Discussed internally: Still not optimal, but ok for now.

**Pull request merged**